src/unsafe-rust/unsafe-traits.md

```diff
@@ -19,7 +19,8 @@ use std::{mem, slice};
 pub unsafe trait IntoBytes {
     fn as_bytes(&self) -> &[u8] {
         let len = mem::size_of_val(self);
-        unsafe { slice::from_raw_parts((&raw const self).cast::<u8>(), len) }
+        let slf: *const Self = self;
```

Comment:
Nit: call it `ptr`?

Reply:
This example is a shortened version of the actual upstream `zerocopy` code, so I think it's best to avoid unnecessary changes.